.github/workflows/build-images-base.yaml

```diff
@@ -202,23 +193,14 @@ jobs:
         if: ${{ steps.cilium-builder-tag-in-repositories.outputs.exists == 'false' }}
         shell: bash
         # To-Do: generate SBOM from source after https://github.com/kubernetes-sigs/bom/issues/202 is fixed
-        # To-Do: format SBOM output to json after cosign v2.0 is released with https://github.com/sigstore/cosign/pull/2479
         run: |
-          bom generate -o sbom_cilium-builder_${{ steps.builder-tag.outputs.tag }}.spdx \
+          bom generate --format json -ao sbom_cilium-builder_${{ steps.builder-tag.outputs.tag }}.spdx.json \
           --image=quay.io/${{ env.QUAY_ORGANIZATION_DEV }}/cilium-builder:${{ steps.builder-tag.outputs.tag }}
 
-      - name: Attach SBOM to Container Image
+      - name: Attest SBOM to Container Image
```

Comment:
`Attest ... to Container Image` -> What does this mean?

Reply:
I tried to meant the workflow is attaching SBOM [attestations](https://github.com/sigstore/cosign/blob/main/specs/ATTESTATION_SPEC.md) to container images.

Reply:
In that case, how about `Attaching attestation to container image` instead?

Reply:
Sounds better. Updated the workflow name.